code/addons/docs/src/preset.ts

```diff
@@ -129,12 +134,15 @@ async function webpack(
   return result;
 }
 
-const docs = (docsOptions: DocsOptions) => {
-  return {
-    ...docsOptions,
+const docs: PresetProperty<'docs'> = (docsOptions, { docs: docsMode }: CLIOptions) => {
+  const result: StorybookConfigRaw['docs'] = {
     defaultName: 'Docs',
-    autodocs: 'tag',
   };
+
+  if (docsOptions && docsMode) {
```

Comment:
Why do we need this?

Reply:
That was already there, I just refactored it.

The big difference is that now I detect addon-docs, by the presence of options.docs, which used to be set by common-preset.

By moving it to addon-docs itself, now `options.docs` becomes `undefined` when `addon-docs` is NOTv set up.